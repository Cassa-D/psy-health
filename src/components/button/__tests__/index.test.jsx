import React from 'react'
import { mount } from 'enzyme'

import Button from '../index'

describe('Render', () => {
  const wrapper = mount(<Button width='100px' height='50px' buttonName='BotÃ£o de teste' fontSize='20px'/>)

  it ('length of children', () => {
    expect(wrapper.find('button').children().find('span')).toHaveLength(14)
  })

  it(' button snapshot', () => {
    expect(wrapper).toMatchSnapshot()  
  })

  it('toBeTruthy', () => {
    expect(wrapper).toBeTruthy()
  })

  
})