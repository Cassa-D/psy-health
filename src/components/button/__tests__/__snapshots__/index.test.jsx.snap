// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render  button snapshot 1`] = `ReactWrapper {}`;
